model_compression_toolkit/qunatizers_infrastructure/pytorch/quantizer_utils.py

```diff
@@ -69,12 +69,47 @@ def fix_range_to_include_zero(range_min: torch.Tensor,
     return min_range_adj, max_range_adj
 
 
+def symmetric_quantizer(tensor_data: torch.Tensor,
```

Comment:
This is not in the correct place; please move it to QAT/PyTorch

Reply:
right. This is true for uniform_quantizer next to it.